rnd/autogpt_server/pyproject.toml

```diff
@@ -36,6 +36,7 @@ anthropic = "^0.25.1"
 
 
 ollama = "^0.3.0"
+feedparser = "^6.0.11"
```

Comment:
poetry lock

Reply:
fixed in [`654a1ef` (#7533)](https://github.com/Significant-Gravitas/AutoGPT/pull/7533/commits/654a1efea0287a65b3fbd4c355875c42c172eacb)